### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by Kamba56**
New html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+
+## Set-up GitHub Actions
+
+Please do the following **steps in this order**:
+
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+   - **Remember** to use the file linked above
+   - **Remember** that `.github` folder starts with a dot.
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
```
Comment (Kamba56) [inline README.md]: I think your README.md file should be descriptive.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      display: flex;
+      flex-direction: row;
+      width: fit-content;
+      justify-content: space-between;
+      align-items: center;
+    }
+
+    nav ul{
+      list-style-type: none;
+      display: flex;
```
Comment (Kamba56) [inline style.css:22]: Please remove unnecessary spaces in your code and add a single space between your selectors and the opening curly braces. This makes your work look neat.

**Review by Kamba56 — COMMENTED** [review]
# Changes Required♻️

Please check my comments below and make the required changes.

Please check your linters for errors.

Please use descriptive commits.

I think you should edit your pull request title and use a more descriptive title and add a summary of all the changes introduced in the pull request.

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @Kamba56,

Good job making use of the GitHub inline comment feature to perform code reviews. 💪💪💪 

FriendsChandlerGIF


Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**